testdata/p4_16_errors_outputs/missing_match.p4-stderr

```diff
@@ -1,3 +1 @@
-missing_match.p4(20): [--Werror=not-found] error: noSuchMatch: declaration not found
-        key = { b : noSuchMatch; }
-                    ^^^^^^^^^^^
+[--Werror=Target model error] error: Declaration of the action NoAction not found; did you include core.p4?
```

Comment:
Should we modify the test program missing_match.p4 so that it gives a similar error message as it did before?  If we do not do that, I worry slightly that we are losing test coverage for the error that this program was exercising before.

Reply:
Done.